### PR TITLE
Add documentation for ws66i core integration

### PR DESCRIPTION
**Pull request by ssaenger**
## Proposed change

This is the documentation for the ws66i core integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/56094
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2809
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by ssaenger** [comment]
Comment to keep PR from going stale.

**Comment by ssaenger** [comment]
@frenck Is it appropriate to contact the vendor of the amplifier and ask for a discount code to be displayed on the documentation page?

**Comment by frenck** [comment]
@ssaenger I would say: No.

File: source/_integrations/ws66i.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Soundavo WS66i 6-Zone Amplifier
+description: Instructions on how to integrate WS66i 6-Zone Home Audio Controller into Home Assistant.
+ha_category:
+  - Media Player
+ha_release: 2022.5
```
Comment (frenck) [inline source/_integrations/ws66i.markdown]:
```suggestion
ha_release: 2022.6
```

File: source/_integrations/ws66i.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ha_config_flow: true
+ha_iot_class: Local Polling
+ha_codeowners:
+  - '@ssaenger'
+ha_domain: ws66i
+ha_platforms:
+  - media_player
+---
+
+The `ws66i` platform allows you to control the [Soundavo Whole-Home Audio Amplifier](https://www.soundavo.com/products/ws-66i) via the local network. This amplifier is an upgrade of the amplifier sold by [Monoprice](https://www.monoprice.com/product?p_id=10761) that adds 2 built-in wireless streamers and an Ethernet port for control over LAN.
```
Comment (frenck) [inline source/_integrations/ws66i.markdown]:
```suggestion
The Soundavo WS66i 6-Zone Amplifier integration allows you to control the [Soundavo Whole-Home Audio Amplifier](https://www.soundavo.com/products/ws-66i) via the local network. This amplifier is an upgrade of the amplifier sold by [Monoprice](https://www.monoprice.com/product?p_id=10761) that adds 2 built-in wireless streamers and an Ethernet port for control over LAN.
```

File: source/_integrations/ws66i.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Source Management
+
+You can configure source names by hitting the **CONFIGURE** button in the integration card.
+
+## Services
+
+### Service `ws66i.snapshot`
+
+Take a snapshot of one or more zones' states. This service, and the following one are useful if you want to play a doorbell or notification sound and resume playback afterward.
```
Comment (frenck) [inline source/_integrations/ws66i.markdown]: We have scenes that provide this functionality in Home Assistant, why does this integration provide its own logic for this?

File: source/_integrations/ws66i.markdown [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+- Power status (On/Off)
+- Mute status (On/Off)
+- Volume level
+- Source
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | no | String or list of strings that point at `entity_id`s of zones.
+
+### Service `ws66i.restore`
```
Comment (frenck) [inline source/_integrations/ws66i.markdown]: Same as above, scenes provide this functionality in Home Assistant, why does this add duplicate logic to Home Assistant?

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by ssaenger** [comment]
@frenck Done. Custom services were removed in https://github.com/home-assistant/core/pull/71717

**Review by frenck — APPROVED** [review]
Thanks, @ssaenger 👍

**Pull request merged**